src/DynamoCore/Logging/AnalyticsService.cs

```diff
@@ -13,7 +13,7 @@ class AnalyticsService
     {
         // Use the Analytics.Core interface so that we do not have to load the ADP assembly at this time.
         private static IAnalyticsUI adpAnalyticsUI;
-
+        private static IAnalyticsClient dynamoAnalyticsClient;
```

Comment:
a bit weird that we need to store the same static instance in 2 places. (on is in DynamoServices proj class Analytics)
Maybe just make the DynamoServices one as internal ?

Reply:
I'll take a look at improving this, yeah, I can make it internal I think.